### DISCUSSION TITLE
Disable GitHub Nx Cloud Report bot comment on pull request

### DISCUSSION THREAD

**Discussion by soanvig** [nrwl/nx#15060] [General]
Is it possible to disable distracting and completely meaningless comment of Nx bot on each pull request?
By publishing a comment it sends notifications that nobody wants.

**Comment by hco** [comment]
I wondered about the same thing - has someone found a solution yet?

We are actually considering dropping nx cloud just because of this.

**Reply by StalkAltan** [reply]
Hi, there is no toggle in the UI to prevent this (though I will mention this to the team). As a workaround, you can set the value of the `NX_BRANCH` environment variable to a nonsense value. Short term this may impact the branch filtering in the Nx Cloud UI, though we are moving away from using this environment variable for that purpose and that should be completed soon.

For others that find this discussion, is there anything that can be added/surfaced in the bot comments to make it more valuable for you? Thanks in advance!

**Reply by estiller** [reply]
I agree there should be an option to remove this comment. It sends a notification no one wants, and links to the NX cloud are available in the "checks" section anyway.

**Reply by soanvig** [reply]
OT: Dropping Nx entirely (not due to that issue, but overall) was one of the best decisions our team made.

**Reply by fregante** [reply]
@StalkAltan please consider adding this, nx-cloud's comments are just noise.

**Reply by jarpoole** [reply]
100% agree. They are driving my team crazy as well...
